src/routes/big_number_card.svelte

```diff
@@ -3,13 +3,22 @@
 
 	export let title: string;
 	export let description: string;
+
+	function updateBackgroundGradient(event: MouseEvent) {
```

Comment:
I believe we should be using arrow functions in the project, instead of function declarations, like this:
```suggestion
	const updateBackgroundGradient = (event: MouseEvent) => {
```
I mostly based my opinion on this due to hoisting. Function declarations are hoisted to the top of the file, which will make testing harder, whenever we need to mock. So in order to keep consistency across the codebase we should settle on the type we use.

The caveat being, I believe this to be standard in React and React Native, if you find Svelte guidelines that say this doesn't matter, happy to revisit my position